tools/helper/template_make_issue_transaction.hpp

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright Soramitsu Co., Ltd. 2016 All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+//
+// Created by Takumi Yamashita on 2017/03/10.
+//
+
+#ifndef TOOLS_ISSUE_TRANSACTION_COMMAND_NAME_C_HPP
+#define TOOLS_ISSUE_TRANSACTION_COMMAND_NAME_C_HPP
+
+#include <iostream>
+#include <vector>
+#include <stdexcept>
+#include <util/logger.hpp>
+#include <transaction_builder/transaction_builder.hpp>
+#include <consensus/connection/connection.hpp>
+#include <infra/config/peer_service_with_json.hpp>
+
+
+namespace tools {
+    namespace issue_transaction {
+
+        namespace COMMAND_NAME_A {
+##CODE
```

Comment:
why not just `#include` directive?
that is very awful solution, it may work but not for so long